common.lic

```diff
@@ -197,9 +197,14 @@ module DRC
   module_function :get_noun
 
   def empty_hands
```

Comment:
It looks like I was wrong and `forage` in common uses this too. How do you feel about merging this as it is?